defaults/network-ents.go

```diff
@@ -77,17 +87,26 @@ func networkNoneEntitlementEnforce(profile *secProfile.Profile) (*secProfile.Pro
  * - Blocked syscalls:
  * 	sethostname, setdomainname, setsockopt(SO_DEBUG)
  */
-func networkUserEntitlementEnforce(profile *secProfile.Profile) (*secProfile.Profile, error) {
+func networkUserEntitlementEnforce(profile secProfile.Profile) (secProfile.Profile, error) {
+	if profile.GetType() != secProfile.OciProfileType {
```

Comment:
can be a followup: feels like all of this profile type checking and type conversion could be factored out to a helper function, ex:
```
func ociProfileConversionCheck(profile secProfile.Profile, entitlementID) (secProfile.OciProfile, error) {
```

since you have this code copied over multiple times